src/samplers/multinomial.jl

```diff
@@ -44,25 +85,26 @@ struct MultinomialSampler <: Sampleable{Multivariate,Discrete}
     alias::AliasTable
 end
 
-MultinomialSampler(n::Int, prob::Vector{Float64}) = 
+MultinomialSampler(n::Int, prob::Vector{Float64}) =
```

Comment:
Could we relax the type constraint from Vector of Float64 to an abstract vector of Real?

Reply:
Done